### ISSUE TITLE
bugs when using list of arrays in labeling locations

### ISSUE THREAD

**Issue by jinzhuyu** [ulfaslak/infostop#17]
In `models.py`

`self._stat_coords = np.around(self._stat_coords / self._min_spacial_resolution) * self._min_spacial_resolution`

If the input data are list of array, we cannot use `self._stat_coords / self._min_spacial_resolution`, correct?

**Comment by ulfaslak** [comment]
Do you mean that `type(self._stat_coords)` would evaluate to `list`? No, that would not work, you can't divide a list with a number in Python.

**Comment by jinzhuyu** [comment]
Yes. If my understanding is correct, then the method "fit_predict" in
models.py is unable to take a list of arrays as input, but in the docstring
for this method, it is stated "data : numpy.array (shape (N, 2)/(N,
3)) or *list
of such numpy.arrays*..."

Thanks!

**Comment by ulfaslak** [comment]
OK I just reviewed the code again. I don't really see the problem here. It DOES work when the input is a list of np.arrays.

Are you getting an error? Can you paste it and show me how to reproduce it?

**Comment by jinzhuyu** [comment]
Thanks. 

I used the following code:
```  

    list_arr = []

    list_arr.append(np.ones([2,2]))

    list_arr.append(np.zeros([2,2]))

    # print(list_arr)

    min_dist_betwn_point = 30

    model_infostop = infostop.SpatialInfomap(r2=min_dist_betwn_point,
                                             label_singleton=True,
                                             min_spacial_resolution=0.0001,
                                             distance_metric='haversine',            
                                             verbose=False)  

    label_list = model_infostop.fit_predict(list_arr)
```

The error message is:

```

    assert data.shape[1] == 2, \

    AttributeError: 'list' object has no attribute 'shape'
```

I believe this error is because the input data is a list of arrays.

Thanks!

jz
